website/package.json

```diff
@@ -20,16 +20,28 @@
     "@docusaurus/core": "3.5.2",
     "@docusaurus/preset-classic": "3.5.2",
     "@mdx-js/react": "^3.0.0",
+    "ajv": "8.17.1",
+    "antd": "5.21.5",
+    "classnames": "^2.0.0",
     "clsx": "^2.0.0",
+    "lodash": "^4.0.0",
+    "lz-string": "1.5.0",
+    "message-box": "0.2.7",
     "prism-react-renderer": "^2.3.0",
+    "raw-loader": "4.0.2",
     "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react-dom": "^18.0.0",
+    "react-feather": "2.0.10",
+    "react-frame-component": "5.2.7",
+    "zod": "3.23.8"
   },
   "devDependencies": {
     "@docusaurus/module-type-aliases": "3.5.2",
     "@docusaurus/theme-classic": "3.5.2",
     "@docusaurus/tsconfig": "3.5.2",
     "@docusaurus/types": "3.5.2",
+    "@types/simpl-schema": "1.12.8",
+    "simpl-schema": "1.13.1",
     "typescript": "~5.5.2"
   },
```

Comment:
The build process fails for now as there are multiple "broken links" (e.g. linking to /playground) due to the fact that this PR migrates the landing page only.

As for the packages, I've added them the landing page didn't require those uniforms packages. I can either add them now or in the next PR where they'll be actually needed.

Reply:
Also, when I've added `"uniforms": "workspace:*"` to the devDependencies in the website app, those two `simpl-schema` packages are still needed. Otherwise I get `Error: Can't resolve 'simpl-schema'`.

Reply:
Ok, let's process with it further later.